[DE] Add `Beatmap` (#7854)
ppy/osu-wiki: wiki/Beatmap/de.md

## Einreichen

*Hauptartikel: [Einreichen](/wiki/Submission)*

Beatmap-Autoren können ihre Kreationen auf die öffentliche [Beatmap-Auflistung](https://osu.ppy.sh/beatmapsets) [hochladen](/wiki/Submission). Auch wenn jede Schöpfung [einer einzelnen Person](/wiki/Beatmap/Beatmap_host) zugeschrieben wird, handelt es sich doch oft um eine gemeinschaftliche Arbeit: einige Schwierigkeitsgrade können von anderen Leuten innerhalb einer [Kollaboration](/wiki/Beatmap/Beatmap_collaborations) oder [separat](/wiki/Beatmap/Guest_difficulty) erstellt werden.

> Nach dem Einreichen erlangt eine Beatmap weitere Felder für Metadaten, wie z. B. Beschreibung, Sprache, Genre und die explizite Inhaltsmarkierung, die von den Beatmap-Autoren selbst auf der Webseite geändert werden kann. Die Beatmap bekommt außerdem einen [Titeltext](/wiki/Beatmap/Title_text), dessen Aussehen mit Hilfe des [Nomination Assessment Teams](/wiki/People/The_Team/Nomination_Assessment_Team) verändert werden kann.

Reviewer: ```suggestion
Nach dem Einreichen erlangt eine Beatmap weitere Felder für Metadaten, wie z. B. Beschreibung, Sprache, Genre und die Markierung für explizite Inhalte, die von den Beatmap-Autoren selbst auf der Webseite geändert werden kann. Die Beatmap bekommt außerdem einen [Titeltext](/wiki/Beatmap/Title_text), dessen Aussehen mit Hilfe des [Nomination Assessment Teams](/wiki/People/The_Team/Nomination_Assessment_Team) verändert werden kann.
```

On first reading this read like "explicit marker for content"

### osu!direct

osu!direct ist die in das Spiel eingebaute Beatmap-Auflistung und wird für den schnellen Zugriff verwendet. Sie bietet eine Reihe an Features ähnlich der Auflistung auf der Webseite. Allerdings ist sie nur für [osu!supporter](/wiki/osu!supporter) verfügbar.

### Beatmap-Pakete

*Hauptartikel: [Beatmap-Pakete](Packs)*

Beatmaps, die im selben Monat gerankt wurden oder dasselbe Thema verfolgen (z. B. derselbe Künstler), werden als Massen-Download in Archive gepackt, die Beatmap-Pakete genannt werden. Die Liste aller offiziellen Pakete ist auf der [Webseite](https://osu.ppy.sh/beatmaps/packs) einsehbar.

### Inoffizielle Quellen

Es gibt auch sekundäre Quellen für Beatmaps, die **vom osu!-Team weder überprüft noch betrieben werden**. Jedoch sind sie in den Foren verfügbar und von der Community geschätzt. Solche Quellen beinhalten:

- Drittanbieter für Beatmap-Spiegel, die auf einer Plattform gehostet werden

> - Pakete und Zusammenstellungen, die von Spielern gemacht und via [P2P-Netzwerke](https://de.wikipedia.org/wiki/Peer-to-Peer) geteilt werden, wie z. B. BitTorrent.

Reviewer: ```suggestion
- Pakete und Zusammenstellungen, die von Spielern gemacht und über [P2P-Netzwerke](https://de.wikipedia.org/wiki/Peer-to-Peer) geteilt werden, wie z. B. BitTorrent.
```

If you used `via` it would have to be `P2P-Netzwerken`

## Beatmaps und Community

### Ranglisten

osu!-Spieler benutzen in allen Spielmodi Beatmaps, um [global im Rang aufzusteigen](/wiki/Performance_points) und [gegeneinander anzutreten](/wiki/Ranking). osu! hat unterschiedliche Arten von beatmapspezifischen Ranglisten, die alle, außer der ersten, nur für [osu!supporter](/wiki/osu!supporter) zur Verfügung stehen:

> - Globale Ranglisten, die in der gesamten aktiven Spielerbasis funktionieren

Reviewer: ```suggestion
- Globale Ranglisten, die die gesamte aktive Spielerbasis verarbeiten
```

Every following bulletpoint describes what players are included in the leaderboard, not what players it "works for" as in who can view them, so I think this is better.

Author: That was a mistranslation on my part. Makes more sense 👍

Author: ```suggestion
- Globale Ranglisten, die die gesamte aktive Spielerbasis miteinbeziehen
```
🤔

Reviewer: 👍

- Globale Ranglisten für jede Kombination an [Spielmodifikationen](/wiki/Game_modifier)
- Länder-spezifische Ranglisten mit Spielern, die dieselben Flaggen tragen
- Freundesranglisten, die die Position eines Spielers auf einer Beatmap verglichen mit seinen Freunden zeigen

Für die 500 besten Scores auf jedem Schwierigkeitsgrad einer Beatmap werden [Replays](/wiki/Gameplay/Replay) angeboten, die entweder online angeschaut oder zur weiteren Anzeige in einer lokalen Rangliste gespeichert werden können.

### Charts und Spotlights

*Hauptartikel: [Beatmap Spotlights](/wiki/Beatmap_Spotlights)*

> Seit ihrem Anfang hatte die Community verschiedene Mittel, um einzigartige und exzellente Beatmaps zu zeigen. Einer der ersten dokumentierten Ansätze waren monatliche und saisonale Charts, die aus einer kleinen Teilmenge an gerankten Beatmaps bestanden. Top-Spieler, die in den Ranglisten Scores in all diesen Beatmaps erreichten, wurden hierbei mit [osu!supporter](/wiki/osu!supporter) belohnt.

Reviewer: ```suggestion
Seit ihrem Anfang hatte die Community verschiedene Mittel, um einzigartige und exzellente Beatmaps zur Schau zu stellen. Einer der ersten dokumentierten Ansätze waren monatliche und saisonale Charts, die aus einer kleinen Teilmenge an gerankten Beatmaps bestanden. Top-Spieler, die in den Ranglisten Scores in all diesen Beatmaps erreichten, wurden hierbei mit [osu!supporter](/wiki/osu!supporter) belohnt.
```

I believe this gets closer to "showcasing"

Das Chart-System entwickelte sich später weiter in das Projekt [Beatmap Spotlights](/wiki/Beatmap_Spotlights), welches jetzt unter ähnlichen Bedingungen betrieben wird, aber stattdessen in Playlists von osu!(lazer) stattfindet.

### Beatmap-Wettbewerbe

*Hauptseite: [Wettbewerbe](/wiki/Contests)*

Die osu!-Community betreibt regelmäßig verschiedene Wettbewerbe, um Kreativität zu fördern und Mapper auszuzeichnen, die ihr Bestes geben. Wettbewerbe reichen von kleinen Wettkämpfen, die in lokalen Gruppierungen ausgetragen werden, bis hin zu großen, die auf globaler Ebene durchgeführt werden und attraktive Preise haben. Gewinner erhalten oft Auszeichnungen in Form von [osu!supporter-Tags](/wiki/osu!supporter) und einem themenbezogenen Profilabzeichen.

### Featured Artists

*Hauptseite: [Featured Artists](/wiki/Featured_Artists)*

> Featured Artists ist ein Community-Programm des [osu!-Teams](/wiki/People/The_Team), das sich auf die Musiklizenzierung von verschiedenen Künstlern konzentriert und die Musik für Mapping verfügbar macht. Die [Liste der Featured Artists](https://osu.ppy.sh/beatmaps/artists) auf der Webseite zeigt jeden beteiligten Künstler und bietet vorgefertigte Templates für das Mapping.

Reviewer: ```suggestion
Featured Artists ist ein Community-Programm des [osu!-Teams](/wiki/People/The_Team), das sich auf die Musiklizenzierung von verschiedenen Künstlern konzentriert und die Musik für Mapping zur Verfügung stellt. Die [Liste der Featured Artists](https://osu.ppy.sh/beatmaps/artists) auf der Webseite zeigt jeden beteiligten Künstler und bietet vorgefertigte Templates für das Mapping.
```
